### PR TITLE
Split docs added, merge docs changes

### PR DESCRIPTION
**Pull request by tehSLy**
`merge` documentation enrichment, `split` documentation added

### CODE REVIEW

**Comment by goodmind** [comment]
`Unit.md` doesn't exists

File: docs/api/effector/merge.md [hunk 1]
```diff
@@ -4,48 +4,71 @@ title: merge
 hide_title: true
 ---
 
-# `merge([eventA, eventB])`
-
-Creates an [event](Event.md), which fires when one of passed events has triggered
+# `merge(unitsArray)`
+Merges any kind of [_`units`_](Unit.md) ([_`Event`_](Event.md), [_`Store`_](Store.md), [_`Effect`_](Effect.md)), resulting single event, which fires upon any of merged units is triggered.
```
Comment (goodmind) [inline docs/api/effector/merge.md]: `Unit.md` doesn't exists

File: docs/api/effector/merge.md [hunk 2]
```diff
@@ -4,48 +4,71 @@ title: merge
...
 
 #### Arguments
 
-1. `events` _(array)_: [events](Event.md) list
+1. `unitsArray` _(Array<[_`Event`_](Event.md)|[_`Store`_](Store.md)|[_`Effect`_](Effect.md)>)_: Array of [_`units`_](Unit.md) to be merged.
```
Comment (goodmind) [inline docs/api/effector/merge.md]: `Unit.md` doesn't exists

File: docs/api/effector/split.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+title: split
+hide_title: true
+---
+
+# `split(event, cases)`
+Pattern matching method, splits event into several, which fire upon source event matches its comparator function.
+
+#### Arguments
+
+1. `event` _([_`Event`_](Event.md))_: Array of [_`units`_](Unit.md) to be merged.
```
Comment (goodmind) [inline docs/api/effector/split.md]: `Unit.md` doesn't exists

**Comment by zerobias** [comment]
Thanks!

**Pull request merged**